CHANGELOG.md

```diff
@@ -97,10 +97,6 @@ This change makes a number of new things possible, e.g.
 - `DoesNotUnderstandError` informs about keyword arguments.
 - Make `Pbind` work with kwargs (this syntax works now: `Pbind(note: Pseq([-3, -2, 0, 1, 6, 13]), dur: 0.05, amp: 0.1).play`) by @JordanHendersonMusic and @telephon in https://github.com/supercollider/supercollider/pull/6530 
 
-### General: Changed
-
-### General: Fixed
-
 ### Class library: Added
```

Comment:
Sorry, I just noticed this - changes in `CHANGELOG.md` should not be part of this PR.

Reply:
OK! Please see: https://github.com/supercollider/supercollider/pull/7079/commits/4ea37d6739b946758eba798396b910bf4ce25de4